components/PageView.vue

```diff
@@ -1,12 +1,12 @@
 <script setup lang="ts">
 const { data } = await useFetch('/api/pageview')
 
-const time = useTimeAgo(computed(() => data.value.startAt))
+const time = useTimeAgo(computed(() => data.value!.startAt))
 </script>
 
 <template>
   <div text-gray:80>
-    <span font-500 text-gray>{{ data.pageview }}</span>
+    <span font-500 text-gray>{{ data?.pageview }}</span>
```

Comment:
```suggestion
    <span font-500 text-gray>{{ data!.pageview }}</span>
```

I think it's better to be consistent

Reply:
> 

done